chart/kubeapps/values.yaml

```diff
@@ -105,6 +105,10 @@ apprepository:
     registry: docker.io
     repository: bitnami/kubeapps-chart-repo
     tag: 1.8.0-r0
+  proxy:
```

Comment:
to differentiate this I would move this within the `initialRepos` key (to avoid confusion with `tiller-proxy`). I would also add a comment explaining what this does and a `enabled` field to enable/disable it:

```yaml
initialRepos:
  # Set env variables for HTTP proxy settings
  proxy:
    enabled: false
    http_proxy: ""
    ...
```

Reply:
Please move this under `initialRepos` instead. If you are setting this within `apprepository` it seems that you are modifying the CRD while this only affects to the initial repositories. Also please add a comment explaining what this does.

Reply:
@andresmgot I can't do that as initialRepos value is an array of repo the key "proxy" will transform initialRepos as a map.

Reply:
you are right, yes then `initialReposProxy` should be fine